README.md

```diff
@@ -64,7 +66,7 @@ $ docker run -t -v `pwd`:/src echidna echidna-test /src/examples/solidity/basic/
 ```
 
 
-If you'd prefer to build from source, use [Stack](https://docs.haskellstack.org/en/stable/README/).
+If you'd prefer to build manually from source, use [Stack](https://docs.haskellstack.org/en/stable/README/).
```

Comment:
no reason to add an extra word here, building from source implies a manual process.